wallet/src/backend.rs

```diff
@@ -556,7 +556,10 @@ impl<'a, Meta: Serialize + DeserializeOwned + Send> CapeWalletBackend<'a>
         // Before the contract can transfer from our account, in accordance with the ERC20 protocol,
         // we have to approve the transfer.
         ERC20::new(erc20_code.clone(), eth.client())
-            .approve(eth.contract.address(), ro.amount.into())
+            .approve(
+                eth.contract.address(),
+                ro.amount.generic_into::<u128>().into(),
```

Comment:
Does `ro.amount.generic_into::<U256>()` not work?

Reply:
It will work with https://github.com/EspressoSystems/cap/pull/50